### PR TITLE
fix compatibility with Nette 3.0.2

### LINKED ISSUE #5: compability with nette/di@3.0.2
new verion nette/di 3.0.2 does not workiging with tester-infrastructure: TypeError: Argument 3 passed to Nette\DI\Resolver::autowireArguments() must be callable, object given, called in /home/vagrant/projects/some-project/vendor/mangoweb/tester-infrastructure/src/MethodArgumentsResolver.php on line 18.

https://github.com/mangoweb-backend/tester-infrastructure/blob/master/src/MethodArgumentsResolver.php#L18

I don't know how fix it. Could you @matej21  look at it and fix it?

### PR DESCRIPTION
**Pull request by petrparolek**
close #5

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	"license": "MIT",
 	"require": {
 		"php": "~7.1",
-		"nette/di": "~2.4 || ~3.0",
+		"nette/di": "~3.0",
```
Comment (JanTvrdik) [inline composer.json]: I think it's better to specify here `~3.0.2 || ~3.1` instead of having the `conflict` section.

Comment (petrparolek) [reply composer.json]: 3.1.x doesn't exist yet

Comment (JanTvrdik) [reply composer.json]: in that case just `~3.0.2` is probably fine

Comment (petrparolek) [reply composer.json]: I thought about it but I was not sure ithis

**Comment by petrparolek** [comment]
fixed PHPStan

File: composer.json (hunk 1 above)
Comment (JanTvrdik) [reply composer.json]: `~3.0` is incorrect. It should be `~3.0.2`

Comment (petrparolek) [reply composer.json]: It is inrevelant because of commit 801a9b849347165107f2b450dda0b1cdc909052b . It is compatible with al version of nette di

**Comment by petrparolek** [comment]
could you merge PR a create new release please?

File: composer.json (hunk 1 above)
Comment (matej21) [reply composer.json]: it looks it is compatible with 2.4, so you can revert changes in composer.json

**Comment by matej21** [comment]
@petrparolek you've accidentally committed nbproject directory.

File: composer.json (hunk 1 above)
Comment (petrparolek) [reply composer.json]: I reveted composer,.json

**Comment by petrparolek** [comment]
yes - fixed

**Comment by matej21** [comment]
merged as ec2b90fa8a417452b08e49a55807e7707d9cb5e6 and published v0.4. thanks

**Comment by petrparolek** [comment]
thanks

**Pull request closed without merging**